### PR TITLE
HGDP + 1KG get call stats for updated samples for the v4 genomes sites HT

### PR DESCRIPTION
**Pull request by jkgoodrich**
Functions copied from this PR: https://github.com/broadinstitute/gnomad_qc/pull/390

This contains the functions required to identify samples that need call stats computed for an update to the v4 genomes sites HT

The first commit is basically a copy from the above PR, and any commits after that are my suggested changes.

### CODE REVIEW

File: gnomad_qc/v4/resources/annotations.py [hunk 1]
```diff
@@ -407,6 +413,32 @@ def get_vrs(
...
+
+    :param test: If True, will return the annotation resource for testing purposes.
+    :param subset: The subset of the HGDP + 1KG/TGP release to return,
+       must be "added", "subtracted", "pop_diff", or "final"
+    :return: MatrixTableResource for specified subset.
+    """
+    # if subset not in ["added", "subtracted", "pop_diff", "final"]:
+    #    raise ValueError(
+    #        "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'final'"
+    #    )
```
Comment (jkgoodrich) [inline gnomad_qc/v4/resources/annotations.py:430]: Sorry, just noticed I have this commented out 
```suggestion
    if subset not in ["added", "subtracted", "pop_diff", "final"]:
        raise ValueError(
            "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'final'"
        )
```

**Review by KoalaQin — CHANGES_REQUESTED** [review]
I only spot a small formatting issue. I went through this twice and tested on drd2 and X gene. Both worked perfectly.

File: gnomad_qc/v4/resources/annotations.py [hunk 2]
```diff
@@ -407,6 +413,32 @@ def get_vrs(
...
+
+    :param test: If True, will return the annotation resource for testing purposes.
+    :param subset: The subset of the HGDP + 1KG/TGP release to return,
+       must be "added", "subtracted", "pop_diff", or "final"
+    :return: MatrixTableResource for specified subset.
+    """
+    if subset not in ["added", "subtracted", "pop_diff", "final"]:
+        raise ValueError(
+            "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'final'"
+        )
```
Comment (KoalaQin) [inline gnomad_qc/v4/resources/annotations.py:430]:
```suggestion
    if subset not in ["added", "subtracted", "pop_diff", "final"]
        raise ValueError(
            "Operation must be one of 'added', 'subtracted', 'pop_diff', or 'join'"
        )
```
The `final` has to be changed to using `get_freq`, right?

**Review by KoalaQin — CHANGES_REQUESTED** [review]
one more thing for the path of the joint freq_ht.

File: gnomad_qc/v4/resources/annotations.py (hunk 2 above)
Comment (jkgoodrich) [reply gnomad_qc/v4/resources/annotations.py:430]: yup, thanks!

**Review by KoalaQin — APPROVED** [review]
LGTM!

**Pull request merged**